nvflare/dashboard/application/blob.py

```diff
@@ -154,6 +154,12 @@ def gen_server(key, first_server=True):
                 "t",
                 exe=True,
             )
+            _write(
+                os.path.join(dest_dir, get_csp_start_script_name("aws")),
+                utils.sh_replace(get_csp_template("aws", "svr", template), {"server_name": entity.name}),
+                "t",
+                exe=True,
+            )
```

Comment:
I notice we put this piece of code under "if not project.ha_mode".
So it means the cloud deployment needs to be done in POC mode?

Reply:
Or how does it work?

Reply:
In cloud deployment, users should use what cloud service provider supports for high availability.  Therefore, we don't want to interfere with those features.

Reply:
got it, thanks for the explanation